test/k8s/suite_test.go

```diff
@@ -76,6 +79,71 @@ var _ = AfterSuite(func() {
 	}
 })
 
+type reportEntry struct {
+	Time  string `yaml:"time"`
+	Name  string `yaml:"name"`
+	Value string `yaml:"value"`
+}
+
+type specReport struct {
+	State         string `yaml:"state"`
+	Description   string `yaml:"description"`
+	ReportEntries []reportEntry
+}
+
+type report struct {
+	Parameters       map[string]string `yaml:"parameters"`
+	SuitePath        string            `yaml:"suitePath"`
+	SuiteDescription string            `yaml:"suiteDescription"`
+	SpecReports      []specReport      `yaml:"specReports"`
+}
+
+var _ = ReportAfterSuite("compile report", func(ginkgoReport Report) {
+	parameters := map[string]string{}
+	for _, envKeyVal := range os.Environ() {
+		if strings.HasPrefix(envKeyVal, "PERF_TEST") {
+			assignment := strings.SplitN(envKeyVal, "=", 2)
+			parameters[assignment[0]] = assignment[1]
+		}
+	}
+
+	report := report{
+		Parameters:       parameters,
+		SuitePath:        ginkgoReport.SuitePath,
+		SuiteDescription: ginkgoReport.SuiteDescription,
+	}
+
+	for _, rep := range ginkgoReport.SpecReports {
+		if rep.LeafNodeType != types.NodeTypeIt {
```

Comment:
Does it include nodes like TableEntry?

Reply:
Yes I believe so, [at this point there's only container vs leaf](https://github.com/onsi/ginkgo/blob/27c2f5d6b2cbdf6ee87cffd6c746466f28787edf/types/types.go#L749)